pkg/artifactory/provider/provider.go

```diff
@@ -183,7 +184,8 @@ func Provider() *schema.Provider {
 			map[string]*schema.Resource{
 				"artifactory_file":     datasource.ArtifactoryFile(),
 				"artifactory_fileinfo": datasource.ArtifactoryFileInfo(),
-				"artifactory_group":    datasource.ArtifactoryGroup(),
+				"artifactory_group":    datasource.DataSourceArtifactoryGroup(),
```

Comment:
Since we are setting package name with the path name (e.g. `datasource_user`), let's update this to `datasource_group` as well.